standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10833,7 +10833,8 @@ public FileMetadataHandler getFileMetadataHandler(FileMetadataExprType type) {
                                                      final String db_name_input,
                                                      final String tbl_name_input)
   throws MetaException, NoSuchObjectException {
-    final String db_name = normalizeIdentifier(db_name_input);
+    final String db_name =
```

Comment:
1. Can we use StringUtils.isNotBlank(db_name_input) instead of (db_name_input!=null). 
2. Also can we have the same check on catName.
3. Can we use unified camel casing naming  convention across variable name.

Reply:
Thanx @ashish-kumar-sharma 
1. I will address.
2. Why for catName? It isn't getting accessed anywhere which can fetch any NPE.
3. I didn't catch this? I haven't introduce any variable itself.

Reply:
@ayushtkn 
1. You haven't introduce any new variable but since you can change the name of other variable as well along with your PR for sanity
2. For catName, dbName and tableName its better to check and normalize everywhere which helps in making sure that hive is case insensitive for name.